src/Components/ExpenseBlock/ExpenseQuestion.js

```diff
@@ -171,7 +171,10 @@ const ExpenseQuestion = ({ expenseData, allExpensesData, setAllExpenses, deleteE
   );
 
   return (
-    <div key={index}>
+    <div
+      key={index}
+      style={{ backgroundColor: index >= 1 && index % 2 !== 0 ? 'var(--light-lavender-background)' : '' }}
```

Comment:
What do you think about conditionally adding a class for background color. Also, I don't think that you need the `index >= 1 &&`.
```suggestion
      style={{ backgroundColor: index % 2 !== 0 ? 'var(--light-lavender-background)' : '' }}
```